cmd/dashboard/main.go

```diff
@@ -76,6 +76,7 @@ func main() {
 	imageNamePrefixTemplate := flag.String("image-name-prefix-template", os.Getenv("NUCLIO_DASHBOARD_IMAGE_NAME_PREFIX_TEMPLATE"), "Go template for the image names prefix")
 	platformAuthorizationMode := flag.String("platform-authorization-mode", defaultPlatformAuthorizationMode, "One of service-account (default) / authorization-header-oidc")
 	dependantImageRegistryURL := flag.String("dependant-image-registry", os.Getenv("NUCLIO_DASHBOARD_DEPENDANT_IMAGE_REGISTRY_URL"), "If passed, replaces base/on-build registry URLs with this value")
+	caCertContent := flag.String("ca-cert-content", "", "Adds ca certificate to git requests")
```

Comment:
please move above, under line 62.
I also suggest to 
1. rename to `git-ca-cert-contents` (it might be chained PEM iirc)
2. use `common.GetEnvOrDefaultString` to allow running nuclio dashboard reading from given envvar